manim/mobject/probability.py

```diff
@@ -157,23 +161,82 @@ def __getitem__(self, index):
 
 
 class BarChart(VGroup):
+    """This is a class for Bar Charts.
+
+    Parameters
+    ----------
+    values
+        The values for the bar chart.
+    height
+        The height of the axes.
+    width
+        The width of the axes.
+    n_ticks
+        Number of ticks.
+    tick_width
+        Width of the ticks.
+    label_y_axis
+        Y axis label
+    y_axis_label_height
+        Height of the label.
+    max_value
+        Maximum value of the data.
+    bar_colors
+        The colors of the bars.
+    bar_fill_opacity
+        The opacity of the bars.
+    bar_stroke_width
+        The stroke width of the bars.
+    bar_names
+        The names of each bar.
+    bar_label_scale_val
+        The label size.
+
+    Examples
+    --------
+    .. manim:: BarChartExample
+
+        class BarChartExample(Scene):
+            def construct(self):
+                lg = [54, 23, 47, 48, 40, 64, 112, 87]
```

Comment:
`lg` is not very explanatory, can you give a more clear, but still short name here?

Reply:
I'll do that.